nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -269,7 +269,8 @@ def add_electrodes(
     recording: se.RecordingExtractor, 
     nwbfile=None, 
     metadata: dict = None,
-    write_scaled: bool = True
+    write_scaled: bool = True,
+    exclude: tuple = ()
```

Comment:
```suggestion
    exclude: Iterable[str]= []
```

Will have to include from `typing import Iterable` above, ofc. Might want to think about naming the option `exclude_property` to make that more explicit, too.

Reply:
Should default to empty tuple, not empty list. Empty lists as default args have [unexpected behavior](https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument)

Reply:
Interesting - did not know that. Modified to empty tuple.

We have exclusion type arguments scattered throughout spikeextractors (now here, of course) and a couple other places that are all list based so I'll check if they do it safely.